packages/next/server/lib/squoosh/impl.ts

```diff
@@ -1,6 +1,32 @@
+import semver from 'next/dist/compiled/semver'
 import { codecs as supportedFormats, preprocessors } from './codecs'
 import ImageData from './image_data'
 
+// Broken in 9.1.269 so hopefully fixed in 9.2.0 :fingerscrossed:
+// See https://crbug.com/1224882
+const V8_FIXED_VERSION = '9.2.0'
+const DELAY_MS = 100
+let _promise: Promise<void> | undefined
+
+function delayOnce(ms: number): Promise<void> {
+  if (!_promise) {
+    _promise = new Promise((resolve) => {
+      setTimeout(resolve, ms)
+    })
+  }
+  return _promise
+}
+
+function maybeDelay(): Promise<void> {
```

Comment:
Should we also detect the worker crashing in `processBuffer` and then fallback to passing the image through with a warning explaining why or does this resolve it fully on the affected v8 versions?

Reply:
I tried isolating the workers in separate forked processes (as opposed to the current threads) in PR #27056 

But that didn't seem to work, it just hangs instead of crashing.